apisix/core/log.lua

```diff
@@ -74,8 +85,9 @@ end
 
 setmetatable(_M, {__index = function(self, cmd)
     local log_level = log_levels[cmd]
-
     local method
+    log_level_update()
+
```

Comment:
There is an issue: if we call the log.xxx in the init phase, we will always generate the method and affect the later phases.

Reply:
Make sense, may be we need a new implement of this.

Reply:
PLTA, now we don't cache the methods generated in the init phase.